### PR TITLE
Bugfix MTE-859 [v114] Attentive mode fix for sync integration tests

### PR DESCRIPTION
**Pull request by clarmso**
[Jira ticket](https://mozilla-hub.atlassian.net/browse/MTE-859)

### Description
[FXIOS-5639](https://mozilla-hub.atlassian.net/browse/FXIOS-5639) introduces changes in the keyboard's behaviour. The keyboard no longer appears when the app starts. All the `app.buttons["urlBar-cancel"].tap()` at the beginning of the tests become unnecessary.

All tests pass on the MacStadium.

### Pull requests checks where applicable
- [ ] Fill in the three TODOs above (tickets number and description of your work)
- [ ] The PR name follows our [naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Accessibility implemented and tested (minimum Dynamic Text and VoiceOver)
- [ ] Unit tests written and passing
- [ ] Documentation / comments for complex code and public methods

### CODE REVIEW

File: Tests/XCUITests/IntegrationTests.swift [hunk 1]
```diff
@@ -189,7 +184,6 @@ class IntegrationTests: BaseTestCase {
     }
 
     func testFxASyncPasswordDesktop () {
```
Comment (isabelrios) [inline Tests/XCUITests/IntegrationTests.swift:186]: This test is failing for me. It cann't find the secureTextField["Passcode field"].. looks like that has changed? I changed L#200 with `let passcodeInput = springboard.secureTextFields.firstMatch` and then it works... 
But if the test passes for you as it is now, it is fine as it is...

**Review by isabelrios — APPROVED** [review]
LGTM, just comment for a test that is still failing for me locally

File: Tests/XCUITests/IntegrationTests.swift (hunk 1 above)
Comment (clarmso) [reply Tests/XCUITests/IntegrationTests.swift:186]: That particular line is failing for me once but passing other times. :-/ I can't reproduce the failure reliably.

**Comment by clarmso** [comment]
@Mergifyio backport release/v113

**Pull request merged**